webapp/src/widget/menu.scss

```diff
@@ -125,8 +125,8 @@
     }
 }
 
-.Menu,
-.SubMenuOption .SubMenu {
+.TodoMenu,
+.SubTodoMenuOption .SubTodoMenu {
```

Comment:
I see changes in the code for `Menu`->`TodoMenu`. Is there any change for `SubMenuOption`->`SubTodoMenuOption` needed?

Reply:
I think it's not needed now. I added it for future use-case.

Reply:
If there is no class name being used in the components `SubTodoMenuOption` or `SubTodoMenu`, then I think they should be removed from the scss file to avoid confusion. Likely, [yagni](https://martinfowler.com/bliki/Yagni.html)

Reply:
@mickmister I removed the unused classnames.